.github/workflows/sub-deploy-integration-tests-gcp.yml

```diff
@@ -435,36 +288,14 @@ jobs:
           --command=' \
           set -ex;
           # Extract the correct disk name based on the device-name
-          export DISK_NAME=$(ls -l /dev/disk/by-id | grep -oE "google-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }} -> ../../[^ ]+" | grep -oE "/[^/]+$" | cut -c 2-); \
-
-          sudo docker run \
-          --name ${{ inputs.test_id }} \
-          --tty \
-          --detach \
-          ${{ inputs.test_variables }} \
-          --mount type=volume,volume-driver=local,volume-opt=device=/dev/$DISK_NAME,volume-opt=type=ext4,dst=${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }} \
-          ${{ vars.GAR_BASE }}/${{ vars.CI_IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }} \
-          '
-
-      # Show debug logs if previous job failed
-      - name: Show debug logs if previous job failed
-        if: ${{ failure() && (inputs.needs_zebra_state && !inputs.needs_lwd_state) && inputs.test_id != 'lwd-full-sync' }}
-        shell: /usr/bin/bash -exo pipefail {0}
-        run: |
-          gcloud compute ssh ${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
-          --zone ${{ vars.GCP_ZONE }} \
-          --ssh-flag="-o ServerAliveInterval=5" \
-          --ssh-flag="-o ConnectionAttempts=20" \
-          --ssh-flag="-o ConnectTimeout=5" \
-          --command=' \
-          lsblk;
-          sudo lsof /dev/$DISK_NAME;
-          sudo dmesg;
-          sudo journalctl -b \
+          DISK_NAME=$(ls -l /dev/disk/by-id | grep -oE "google-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }} -> ../../[^ ]+" | grep -oE "/[^/]+$" | cut -c 2-);
+          sudo mkfs.ext4 -v /dev/$DISK_NAME \
```

Comment:
This change might need a full sync test, I think the fastest test that uses this script is a testnet full sync.

Reply:
This behavior is already in the `main` branch: https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/deploy-gcp-tests.yml#L196-L197

Reply:
By the way, I can't trigger a sync test unless we merge (as these are "new" workflows) https://github.com/ZcashFoundation/zebra/actions/workflows/ci-integration-tests-gcp.yml

Reply:
So do we want to delay this large CI change until after the release?
I'll leave it to you and @upbqdn to decide, because he is the release manager.

Reply:
I'd prefer not to as this PR won't have an impact on the built image. But I don't mind if @upbqdn prefers not to include this PR; that's ok for me as we can merge it just after the release. We can have it approved and labeled to not merge. Whatever is less traumatic.

Reply:
My concern was more about unexpected CI failures delaying the release, but I also think we've done good testing here.

Reply:
This release will be big enough even without this change, so let's delay?